fedmsg.d/pdcupdater-example.py

```diff
@@ -30,6 +30,9 @@
     # PkgDB details
     'pdcupdater.pkgdb_url': 'https://admin.fedoraproject.org/pkgdb',
 
+    # MBS details
```

Comment:
Note we will have to configure this in Ansible

Reply:
Ack!